src/Akka.Persistence.PostgreSql/Journal/PostgreSqlQueryExecutor.cs

```diff
@@ -55,7 +53,14 @@ public PostgreSqlQueryExecutor(PostgreSqlQueryConfiguration configuration, Akka.
                     CONSTRAINT {Configuration.MetaTableName}_pk PRIMARY KEY ({Configuration.PersistenceIdColumnName}, {Configuration.SequenceNrColumnName})
                 );";
 
-            switch (_configuration.StoredAs)
+            HighestSequenceNrSql = $@"
```

Comment:
It's strange, because it looks like the same SQL statement is used by default. See default [QueryExecutor](https://github.com/akkadotnet/akka.net/blob/d2561620037944190f4381006133a87304536548/src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs#L324).

Reply:
So this PR does nothing?

Reply:
If current Postgres implementation will be up-to-date with current akka.persistence.sql.common, I guess it won't change anything.

Reply:
```csharp
            HighestSequenceNrSql = $@"
                SELECT MAX(u.SeqNr) as SequenceNr 
                FROM (
                    SELECT e.{Configuration.SequenceNrColumnName} as SeqNr FROM {Configuration.FullJournalTableName} e WHERE e.{Configuration.PersistenceIdColumnName} = @PersistenceId
                    UNION
                    SELECT m.{Configuration.SequenceNrColumnName} as SeqNr FROM {Configuration.FullMetaTableName} m WHERE m.{Configuration.PersistenceIdColumnName} = @PersistenceId) as u";
```

This is the default. This PR does MAX inside of the unions as well.